docs/manuals/gui/viselements/table.md_template

```diff
@@ -549,6 +549,45 @@ in the control definition:
         ...
         tgb.table("{data}", style="{lambda s, idx, r: 'blue-cell' if idx % 2 == 0 else 'red-cell'}" />
         ```
+## Styling column
+
+To style a specific column, you need to modify the example codes of [Styling rows](#styling-rows) 
+
+Here is how a column styling function can be defined:
+
+```python
+def even_odd_style(_1, _2, index, _3, _4, _5):
```

Comment:
The number of parameters is incorrect. I think you should name the parameters for people to copy and paste easily

```python
def even_odd_style(state, value, index, rown, column_name):
...

```